substrate/bin/node/runtime/src/lib.rs

```diff
@@ -591,8 +591,8 @@ pallet_staking_reward_curve::build! {
 
 parameter_types! {
 	pub const SessionsPerEra: sp_staking::SessionIndex = 6;
-	pub const BondingDuration: sp_staking::EraIndex = 24 * 28;
-	pub const SlashDeferDuration: sp_staking::EraIndex = 24 * 7; // 1/4 the bonding duration.
+	pub const BondingDuration: sp_staking::EraIndex = 28 * 24 / 6; // 28 days; Era is 6h long
```

Comment:
This never was 6 months though? Unstaking was supposed to go from 6 months to 1 month. I d expect to see a change of deleting the 6 months and replacing with 1 but i see just some minor math changes?

Reply:
It was 24*28 Eras before. Era is 6h long, so it was 28*6 days = 6 months (if yoy define month as 4 weeks)
Now its 28*24/6 eras, so 6x shorter, so 1 month. Or, given we have 4 eras every day, its 28 * 24 / 6 / 4 = 28 days long.